AdminNotificationManagerPlugin.inc.php

```diff
@@ -146,19 +148,31 @@ function getTemplatePath($inCore = false) {
 	 * @return array
 	 */
 	function _getAdminList() {
-		$user = null;
+		$arrayOfContexts = $this->_getContexts();
 		$users = array();
+		$userId = array();
+		$user = null;
+		$userFactory = new PKPUserService();
 
-		$roleDAO = DAORegistry::getDAO('RoleDAO');
-		$userDAO = $roleDAO->getUsersByRoleId(ROLE_ID_SITE_ADMIN);
+		foreach($arrayOfContexts as $context) {
+		$userGroupDAO = DAORegistry::getDAO('UserGroupDAO');
```

Comment:
The lines within the `foreach` block should be consistently indented.  Here this and following line are outdented one level too far.